code/src/components/ThoughtMessage.css

```diff
@@ -0,0 +1,81 @@
+article{
+  background-color:#ffffff;
+  width: 85%;
+  margin: 20px 0;
+  padding: 15px;
+  border: darkgray 2px solid;
+  border-radius: 2px;
+  box-shadow: black 8px 8px;
+  word-wrap: break-word;
+}
+
+.wrapper {
```

Comment:
You have done such great job naming your classes, could this one also be specified some more like thought-wrapper or article-wrapper or  something_just for consistency